play.pokemonshowdown.com/src/panel-battle.tsx

```diff
@@ -758,6 +759,69 @@ class BattlePanel extends PSRoomPanel<BattleRoom> {
 		}
 		return null;
 	}
+
+	renderAfterBattleControls() {
+		const room = this.props.room;
+		return <div class="controls">
+			<p>
+				<span style="float: right"><a
+					onClick={this.handleDownloadReplay}
+					href={`//${Config.routes.replays}/`}
+					class="button replayDownloadButton"
+				>
+					<i class="fa fa-download"></i> Download replay</a>
+				<br />
+				<br />
+				<button class="button" name="cmd" value="/savereplay">
+					<i class="fa fa-upload"></i> Upload and share replay
+				</button>
+				</span>
+
+				<button
+					class="button" name="instantReplay" onClick={this.handleInstantReplay}
+				>
```

Comment:
```suggestion
				<button class="button" onClick={this.handleInstantReplay}>
```

You don't need `name` if you have an `onClick` handler.